.markdownlint.json

```diff
@@ -0,0 +1,6 @@
+// Rule configurations for markdownlint VS Code extenstion
+{
```

Comment:
Is this just controlling the markdown linter for Vscode? I haven't seen this before

Reply:
yup, I considered dropping this. but since nothing in this repo is required, it seemed worth showing how this can be done and which rules jumped out as ones we're likely to break often/won't care about